PyRoute/Star.py

```diff
@@ -342,17 +342,24 @@ def sec_pos(self, sector):
     def set_location(self, dx, dy):
         # convert odd-q offset to cube
         q = int(self.position[0:2]) + dx - 1
-        r = int(self.position[2:4]) + dy - 1
+        raw_r_offset = 41 - int(self.position[2:4])
```

Comment:
What does this make as the zero point for the positions?

I'd love to have a (short) documentation comment here about what the sector (dx/dy) and hex position is converted to by this method. For example: 

Core 0101 which is sector position 0,0, and hex position 01, 01 becomes q=0, r=0, and x=0, y=0, z=0 , with coreward being positive, rimward being negative X values, trailing being positive, spinward being negative Y values. 

(Please correct the above before including it)

Reply:
Hilariously enough, @tjoneslo , the origin ends up as Reference (Core 0140).  Same as in-game, but by a different path.

Lemme think up an explanatory blurb.

Reply:
@tjoneslo , have sent up a comment.  This should now be good to merge